src/onegov/town6/locale/de_CH/LC_MESSAGES/onegov.town6.po

```diff
@@ -2560,7 +2587,7 @@ msgid "Analytics"
 msgstr "Web-Statistik"
 
 msgid "Holidays"
-msgstr "Feiertage / Schulferien"
+msgstr "Feiertage"
```

Comment:
There's definitely some things that went wrong during the merge here. This has undone a bunch of work I did to make sure org and town6 translations match again. There were a handful of translations that were different and clearly wrong/outdated in one of the two modules.

`git checkout` the po files from master and copy the new translations back in, so we don't regress the translations again.